src/RequestFeedback.cc

```diff
@@ -180,7 +180,7 @@ std::string SolverRequester::Feedback::asUserString( const SolverRequester::Opti
       const std::string & pname( pIdent( _objsel ) );
       return( str::Format(_("Patch '%1%' is optional. Use '%2%' to install it, or '%3%' to include all optional patches."))
 	      % pname
-	      % ( "zypper in " + pname )
+	      % ( "zypper in -t patch" + pname )
```

Comment:
Please change it to `% ( "zypper in patch:" + _objsel->name() )` to be consisent with other cases like
```
    case PATCH_UNWANTED:
    {
      const std::string & pname( pIdent( _objsel ) );
      std::string cmd1 = "zypper in --force patch:" + _objsel->name();
      std::string cmd2 = "zypper rl patch:" + _objsel->name();
```